phpBB/phpbb/mention/controller/UserMentionController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ *
+ * phpBB mentions. An extension for the phpBB Forum Software package.
+ *
+ * @copyright (c) 2016, paul999, https://www.phpbbextensions.io
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ */
+
+	namespace phpbb\mention\controller;
+
+	use phpbb\db\driver\driver_interface;
+	use phpbb\request\request_interface;
+	use Symfony\Component\HttpFoundation\JsonResponse;
+	use phpbb\controller\helper;
+	/**
+	 * phpBB mentions main controller.
+	 */
+
+	class UserMentionController {
```

Comment:
Please rename your class as per the coding guidelines; http://area51.phpbb.com/docs/master/coding-guidelines.html#namingvars